HomeTasks/AlgorithmsAndLoops/TaskUtilities.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace AlgorithmsAndLoops
+{
+    public static class TaskUtilities
+    {
+        public static int[] GetDigits(int number)
+        {
+            var inversedDigits = new List<int>();
+
+            int divider = 1;
+            int modDivider = 10;
+
+            var lastModDivision = number - 1;
+            while (lastModDivision != number)
+            {
+                lastModDivision = number % modDivider;
+                inversedDigits.Add(lastModDivision / divider);
+                modDivider *= 10;
+                divider *= 10;
+            }
+
+            var digits = new int[inversedDigits.Count];
+
+            for (int i = 0; i < digits.Length; i++)
+            {
+                digits[i] = inversedDigits[digits.Length - i - 1];
+            }
+
+            return digits;
+        }
+
+        public static int[] GenerateArray(int length)
+        {
+            int[] array = new int[length];
```

Comment:
Do not forget to separate logic parts by an empty line

Reply:
Under "logic parts" you mean loops, "if-else", "switch", "using", class members and so on?